drivers/adc/Kconfig.ads1x1x

```diff
@@ -18,6 +18,10 @@ config ADC_ADS1X1X
 
 if ADC_ADS1X1X
 
+config ADC_ADS1X1X_TRIGGER
+	bool
+	default y
```

Comment:
This should be conditional on the property being configured via Devicetree

Reply:
I will try to make some time to add this, hopefully next week

Reply:
I have introduced a new commit with a new device tree  macro `DT_ANY_COMPAT_HAS_PROP_STATUS_OKAY` 

This has been added at the top of the file

```
#if DT_ANY_COMPAT_HAS_PROP_STATUS_OKAY(ti_ads1115, alert_rdy_gpios) || \
    DT_ANY_COMPAT_HAS_PROP_STATUS_OKAY(ti_ads1114, alert_rdy_gpios) || \
    DT_ANY_COMPAT_HAS_PROP_STATUS_OKAY(ti_ads1113, alert_rdy_gpios) || \
    DT_ANY_COMPAT_HAS_PROP_STATUS_OKAY(ti_ads1014, alert_rdy_gpios) || \
    DT_ANY_COMPAT_HAS_PROP_STATUS_OKAY(ti_ads1013, alert_rdy_gpios)

#define ADC_ADS1X1X_TRIGGER

#endif
```
CONFIG_ADC_ADS1X1X_TRIGGER has been removed and replaced with `ADC_ADS1X1X_TRIGGER`